Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -12,233 +11,234 @@
+    {
+        // For when the plugin being previously disabled and is now reenabled
+        if (!_initialized)
         {
-            // For when the plugin being previously disabled and is now renabled 
-            if (!initialized)
-            {
-                LoadBookmarksIfEnabled();
-            }
+            LoadBookmarksIfEnabled();
+        }
 
-            string param = query.Search.TrimStart();
+        string param = query.Search.TrimStart();
 
-            // Should top results be returned? (true if no search parameters have been passed)
-            var topResults = string.IsNullOrEmpty(param);
+        // Should top results be returned? (true if no search parameters have been passed)
+        var topResults = string.IsNullOrEmpty(param);
 
 
-            if (!topResults)
-            {
-                // Since we mixed chrome and firefox bookmarks, we should order them again                
-                var returnList = cachedBookmarks.Select(c => new Result()
-                {
-                    Title = c.Name,
-                    SubTitle = c.Url,
-                    IcoPath = @"Images\bookmark.png",
-                    Score = BookmarkLoader.MatchProgram(c, param).Score,
-                    Action = _ =>
+        if (!topResults)
+        {
+            // Since we mixed chrome and firefox bookmarks, we should order them again
+            return _cachedBookmarks
+                .Select(
+                    c => new Result
                     {
-                        context.API.OpenUrl(c.Url);
+                        Title = c.Name,
+                        SubTitle = c.Url,
+                        IcoPath = @"Images\bookmark.png",
+                        Score = BookmarkLoader.MatchProgram(c, param).Score,
+                        Action = _ =>
+                        {
+                            _context.API.OpenUrl(c.Url);
 
-                        return true;
-                    },
-                    ContextData = new BookmarkAttributes
-                    {
-                        Url = c.Url
+                            return true;
+                        },
+                        ContextData = new BookmarkAttributes { Url = c.Url }
                     }
-                }).Where(r => r.Score > 0);
-                return returnList.ToList();
-            }
-            else
-            {
-                return cachedBookmarks.Select(c => new Result()
-                {
-                    Title = c.Name,
-                    SubTitle = c.Url,
-                    IcoPath = @"Images\bookmark.png",
-                    Score = 5,
-                    Action = _ =>
-                    {
-                        context.API.OpenUrl(c.Url);
-                        return true;
-                    },
-                    ContextData = new BookmarkAttributes
+                )
+                .Where(r => r.Score > 0)
+                .ToList();
```

Comment:
Same as in `FirefoxBookmarkReader.cs`: I think it's much more readable when every chained method call is in a separate line, and I'd like to hear your opinion on this.